desktop/dev-environments.md

```diff
@@ -80,7 +80,7 @@ Docker then clones the repository with your specified branch or tag.
 
 > **Note**
 >
-> Known issue: when cloning a Git branch using `https://`, you must remove the `.git` suffix at the end of the URL.
+> Known issue: With versions prior Desktop 3.6.0, when cloning a Git branch using `https://`, you must remove the `.git` suffix at the end of the URL.
```

Comment:
Should we remove this note totally now this is fixed ? (or put it at the bottom with other notes ?)

Reply:
Perhaps something like

```suggestion
> Make sure the URL does not have a `.git` suffix if you are cloning a Git branch
> using `https://` and are using Docker Desktop prior to 3.6.0.
```

Yes, it would be good to keep "no longer" issues out of the main docs (after all, we'd like people to use a current version).

Perhaps add a "known issues" section in the v3.5 release notes for Mac and Windows, and move it there? https://github.com/docker/docker.github.io/blob/master/docker-for-mac/release-notes.md
https://github.com/docker/docker.github.io/blob/master/docker-for-windows/release-notes.md

And be sure to add a note to the 3.6 release notes that it's fixed.
3.6 has not been release yet though, so we need to wait for that (or combine this change with the release notes PR for that?)


That said; the example does not show a `.git` suffix in the `https://` example; do we need the note at all? (we could consider it a user error if they added `.git`?

Does the `git@` example also work if you remove the `.git` suffix? If so, perhaps remove the suffix there as well to prevent users accidentally adding one.

Reply:
Wondering; nothing in the example is _pushing_ too GitHub. Should we remove all uses of `git@` and replace them with `https://` ? Doing so won't require users to set up ssh, and makes sure the tutorial "just works"

If we _do_ want to document how to work with ssh authentication, I think we should move that to a separate ("advanced") section, so that the "main" tutorial is simple to use, and gives the "it works!" experience.,  and worry about authentication later.

Reply:
Indeed, 3.6 has not been released yet. 
Makes sense to add a note in 3.6 release that this is fixed. I think we can then remove it from the main doc (esp on a preview, and more a side feature, not totally the main path)

Reply:
@gtardif @rumpl any ideas on using `https://` in the examples on this page to simplify things? (see above)

Reply:
I agree yes, I think I saw somewhere that people use https more than ssh when they use git.

Reply:
Yes, `https://` is quite common, and has some advantages (mostly ease of use, and less chance of corporate firewalls blocking requests); https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_http_protocols

We should have some info on how to use dev-containers with `git` / `ssh` ("advanced" use), but if we don't need it for the first introduction, my vote would be to keep things easy for the user.